classes/dev/ComposerScript.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
We've decided to not add `strict_types` anymore (maybe the existing ones will be removed later).
I don't think you need to change anything here (i.e. it's just a script), so I'm just sharing the information.

Reply:
Thanks a lot @jonasraoni! I removed it and will merge once the tests pass...